samples/bluetooth/broadcast_audio_source/prj.conf

```diff
@@ -13,5 +13,3 @@ CONFIG_BT_ISO_TX_BUF_COUNT=4
 CONFIG_BT_ISO_TX_MTU=60
 
 CONFIG_BT_DEVICE_NAME="Broadcast Audio Source"
-
-CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
1) this seems to not be needed anymore
2) If enabled, it also now gets disabled by unfulfilled dependency on BT_ECC - and if enabled, we seem to not be able to have encrypted broadcast working)

=> Removing the BT_TINYCRYPT_ECC config